#!/usr/bin/env python2
PACKAGE = "event_simulator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("canny_low_thresh", int_t, 0, "Lower threshold for canny edge detector", 120, 1, 1000)
gen.add("canny_high_thresh", int_t, 0, "pixel array size (y)", 150, 1, 1000)
gen.add("canny_kernel_size", int_t, 0, "Kernel size for canny edge detector", 3, 1, 11)

exit(gen.generate(PACKAGE, "event_simulator", "EventSimCfg"))
